src/Domain/User.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Domain;
+
+use App\Domain\UserRole;
+
+class User
+{
+    private ?int $id;
```

Comment:
Property must not be accessed before initialization
```suggestion
    private ?int $id = null;
```